deployment/chainloop/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: Chainloop is an open source software supply chain control plane, a
 
 type: application
 # Bump the patch (not minor, not major) version on each change in the Chart Source code
-version: 1.86.14
+version: 1.86.17
```

Comment:
make sure you bump this compared to the one in main.

Reply:
Sure, local one went rogue, sorry for that.